Cubical/Codata/Conat/Base.agda

```diff
@@ -0,0 +1,13 @@
+{-# OPTIONS --cubical --safe --guardedness #-}
+module Cubical.Codata.Conat.Base where
+
+open import Cubical.Core.Everything
+
+data Pred (A : Set) : Set where
```

Comment:
Is `pred` a good name here? It's short for `predecessor`, but you write `zero = conat pzero`, and the predecessor of 0 is not 0... the predecessor function here is `\ x -> unwrap-pred (pred x)`, as shown by  `unwrap-pred (pred two) ≡ one`...

Reply:
Do you have any recommendations? I agree with you but I failed to find a better name.

Reply:
Renamed.